lib/guide/index.js

```diff
@@ -1,11 +1,14 @@
 /*jshint node:true*/
-const EngineAddon = require('ember-engines/lib/engine-addon');
+const EngineAddon = require("ember-engines/lib/engine-addon");
 module.exports = EngineAddon.extend({
```

Comment:
These formatting changes seem unrelated, can you revert them?

Reply:
Whoops! Yes, sorry – thought I'd caught and reverted all of those already!

Reply:
Done and force pushed.